### DISCUSSION TITLE
How to remove gaps between dots ?

### DISCUSSION THREAD

**Discussion by ugnius1994** [chillerlan/php-qrcode#229] [Q&A]
![new](https://github.com/chillerlan/php-qrcode/assets/46896856/4784f4cd-1df9-45a4-8103-c7e1de2b4242)

 $options = new QROptions([
        'eccLevel' => EccLevel::L, // Consistent ECC level
        'scale' => 15, // Consistent scale for all QR codes
        'outputType' => QROutputInterface::GDIMAGE_PNG,
        'outputBase64' => false,
        'imageTransparent' => true,
        'drawCircularModules' => true,
        'addQuietzone' => false,
        'drawLightModules' => true,
        'circleRadius' => 0.4,
    ]);

**Comment by ugnius1994** [comment]
it should be same like this 
![image](https://github.com/chillerlan/php-qrcode/assets/46896856/22c341d9-7c8e-49eb-a826-0c44c90ddaeb)

**Reply by codemasher** [reply]
I think what you are looking for is the `circleRadius` setting? If you set it to `0.5`, the dots will touch (the diameter is 1 module).

**Comment by ugnius1994** [comment]
and how make more dots like i need extra 20

**Reply by codemasher** [reply]
Could you please give an example, i don't know what you mean exactly.
